src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java

```diff
@@ -48,13 +53,24 @@ static ZipInputStream noOpVerification(ZipStreamWithSignature zipWithSignature)
     static ZipInputStream sha256WithRsaVerification(ZipStreamWithSignature zipWithSignature) {
         Map<String, byte[]> zipEntries = extractZipEntries(zipWithSignature.zipInputStream);
         if (!verifyFileNames(zipEntries)) {
-            throw new SignatureValidationException(
-                "Zip entries do not match expected file names. "
-                + "Actual names = " + zipEntries.keySet()
+            log.warn("Signature Failure. Zip entries do not match expected file names. "
+                + "Container = {} - File = {} - Actual names = {}",
+                zipWithSignature.container, zipWithSignature.zipFileName, zipEntries.keySet()
+            );
+            throw new DocSignatureFailureException(
```

Comment:
should we be doing something with these files?
won't be keep trying to process them?

Reply:
That is not included in this story, but we should definitely get all the "garbage" out of the way :-)

Reply:
Is this the correct exception? Was signature even checked?

Reply:
That's part of the specification for signature verification. The file certainly cannot be verified so it's clearer to use the same exception.